app/DTO/LycheeGitInfo.php

```diff
@@ -8,7 +8,7 @@ class LycheeGitInfo extends DTO
 	public ?string $commit;
 	public ?string $additional;
 
-	public function __construct(string $branch, ?string $commit = null, ?string $additional = null)
+	public function __construct(string $branch, ?string $commit = null, string $additional = null)
```

Comment:
```suggestion
	public function __construct(string $branch, ?string $commit = null, ?string $additional = null)
```

Removing the question mark operator from the type does not make sense if the default is still `null`.